src/styles/breakpoints.ts

```diff
@@ -1,10 +1,10 @@
 import { BREAKPOINT_REM } from '@/constants/page';
 
 export default {
-  mobile: `(max-width: ${BREAKPOINT_REM.mobile})`,
-  notMobile: `(min-width: ${BREAKPOINT_REM.mobile})`,
-  tablet: `(max-width: ${BREAKPOINT_REM.tablet})`,
-  notTablet: `(min-width: ${BREAKPOINT_REM.tablet})`,
+  mobile: `(width <= ${BREAKPOINT_REM.mobile})`,
```

Comment:
is this valid css? amazing

Reply:
yup yup. turns out the min/max-width are both inclusive so you can end up in a place where tablet and notTablet are true, which makes things weird.